readme.md

```diff
@@ -115,7 +115,6 @@ If you want to contribute, please read the [contribution guidelines](contributin
 - [Node](https://github.com/mysticatea/eslint-plugin-node) - Additional ESLint's rules for Node.js.
 - [Notice](https://github.com/nickdeis/eslint-plugin-notice) - An eslint rule that checks the top of files and fixes them too!
 - [Optimize Regex](https://github.com/BrainMaestro/eslint-plugin-optimize-regex) - Optimize regex literals.
-- [SonarJS](https://github.com/SonarSource/eslint-plugin-sonarjs) - Rules detecting bugs and suspicious patterns.
```

Comment:
While I think this PR is a very good idea, I think the reference made here previously to "suspicious" may be misleading in suggesting the rules are about security; instead, I think what is meant here is patterns that are likely to be problems (e.g., duplicate `else if` blocks).

Reply:
I once had an external security audit, tell me I should be using this rule set for security reasons.

Reply:
I think it would simply because it catches bad practices. Do you see any specific rules in SonarJS which are directly security related? There are other rules in the section which check potential errors and best practices too, so I think the security section should be limited to those which are indeed more closely related to security.

Reply:
I could go either way on this one, I'll let @dustinspecker make the call

Reply:
A security section is a great idea. I agree with @brettz9. Let's keep SonarJS where it is for now. I didn't find anything that seemed related to security in SonarJS to me.

Reply:
Moved back